<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="hlCentral">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="vlMain">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hlCenter" stretch="0">
        <item>
         <widget class="QTabWidget" name="tabDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
QTabBar::tab {
/*
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
	*/
	font-family: Calibri;
	font-style: normal;
	font-size: 12pt;
	font-weight: bold;

}

	QTabBar::tab { height: 25px; width:250px; }

QTabBar::tab:selected {
	color : blue;

}


QTabBar::tab:!selected {
   color : black;
}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabImage">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(122, 122, 122);</string>
           </property>
           <attribute name="title">
            <string>EXPLORATEUR DES PHOTOS</string>
           </attribute>
           <layout class="QVBoxLayout" name="vlImage"/>
          </widget>
          <widget class="QWidget" name="tabPreview">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(122,122,122);</string>
           </property>
           <attribute name="title">
            <string>APERCU DU PDF</string>
           </attribute>
           <layout class="QVBoxLayout" name="vlPreview"/>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlBottomBar" stretch="0,0,5,0">
        <item>
         <widget class="QLabel" name="laProfile">
          <property name="text">
           <string>Profil :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="laProfileName">
          <property name="text">
           <string>défaut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="pbLoading">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="laLoading">
          <property name="text">
           <string>Prêt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionLoadProfile"/>
    <addaction name="actionSaveProfile"/>
    <addaction name="actionSaveProfileTo"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDefault"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionOnlineDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionDonate"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOnlineDocumentation">
   <property name="text">
    <string>Tutoriel</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionSaveProfileTo">
   <property name="text">
    <string>Sauvegarder profil sous...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionSetDefaultProfile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Définir profil par défaut</string>
   </property>
  </action>
  <action name="actionLoadProfile">
   <property name="text">
    <string>Charger profil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProfile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauvegarder profil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Réinitialiser profil</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Faire un don</string>
   </property>
  </action>
  <action name="actionSignaler_un_bug">
   <property name="text">
    <string>Signaler un bug</string>
   </property>
  </action>
  <action name="actionSaveDefault">
   <property name="text">
    <string>Sauvegarder profil par défaut</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
